fe/fe-core/src/main/java/com/starrocks/load/loadv2/SparkLoadJob.java

```diff
@@ -588,7 +589,7 @@ backend, new Replica(tabletId, backend.getId(), -1, NORMAL),
                     String errMsg = new LogBuilder(LogKey.LOAD_JOB, id)
                             .add("database_id", dbId)
                             .add("label", label)
-                            .add("error_msg", "all partitions have no load data")
+                            .add("error_msg", "No partitions have data available for loading")
```

Comment:
It's better to move it into ErrorCode.java too

Reply:
`empty_load_as_error` is not supported in spark load